plugins/test/dynamic_test.cc

```diff
@@ -280,19 +290,35 @@ void DynamicTest::BenchHttpHandlers(benchmark::State& state) {
     BM_RETURN_IF_ERROR(headers.status());
     response_headers = *headers;
   }
+  std::vector<std::string> request_body_chunks;
+  for (const auto& request_body : cfg_.request_body()) {
+    request_body_chunks.emplace_back(request_body.input().content());
+  }
+  std::vector<std::string> response_body_chunks;
+  for (const auto& response_body : cfg_.response_body()) {
+    response_body_chunks.emplace_back(response_body.input().content());
+  }
   for (auto _ : state) {
     if (request_headers) {
```

Comment:
Don't you need to move `auto stream = TestHttpContext(handle);` inside the loop?

Reply:
https://github.com/GoogleCloudPlatform/service-extensions-samples/pull/95

Reply:
After looking at your other comment, I think it would be easier to include this fix in this PR.